test/common/run-tests

```diff
@@ -285,7 +285,7 @@ def run(opts, image):
                     # restart it to avoid an unbounded number of test retries and follow-up errors
                     if not opts.machine and (poll_result == 124 or (retry_reason and b"test harness" in retry_reason)):
                         # try hard to keep the test output consistent
-                        testlib.MachineCase.kill_global_machine()
+                        testlib.MachineCase.kill_global_machine() # TODO?
```

Comment:
Let's replace this with `# FIXME: May change the ports when a parallel run-tests is running"